app/assets/javascripts/discourse/app/instance-initializers/admin-sidebar.js

```diff
@@ -172,16 +179,49 @@ export function buildAdminSidebar(navConfig) {
   });
 }
 
+export function addAdminSidebarSectionLink(sectionName, link) {
+  if (!additionalAdminSidebarSectionLinks.hasOwnProperty(sectionName)) {
+    additionalAdminSidebarSectionLinks[sectionName] = [];
+  }
+
+  // make the name extra-unique
+  link.name = `admin_additional_${sectionName}_${link.name}`;
+
+  // href or route needs to be provided
+  if (!link.href && !link.route) {
+    return;
+  }
+
+  // label must be valid, don't want broken [XYZ translation missing]
+  if (!I18n.lookup(link.label)) {
+    return;
+  }
```

Comment:
I wonder if we should at least log a warning in the console instead of returning silently.

Reply:
Yep sure I will add some debug logging :+1: